examples/size_of_variables.c

```diff
@@ -0,0 +1,15 @@
+#include "../czech.h"
+#include <stdio.h>
+
+číslo hlavní(prázdno) {
+    číslo čísloTyp;
+    plav plavTyp;
+    dvojitý dvojitýTyp;
+    znak znakTyp;
+    // Velikost operator is used to evaluate the size of a variable
+    tisknif("Size of číslo: %ld bytes\n", velikost(čísloTyp));
+    tisknif("Size of plav: %ld bytes\n", velikost(plavTyp));
+    tisknif("Size of dvojitý: %ld bytes\n", velikost(dvojitýTyp));
+    tisknif("Size of znak: %ld byte\n", velikost(znakTyp));
```

Comment:
size of v č? 🤔

Reply:
Popisy po anglicky, aby sa mohol každý naučiť v Č 😁

Reply:
když se chce naučit Č, tak se podívá na #define ...
ale takhle se může přiučit i něco z čestiny :)

Reply:
Čiže to dám po česky všetko ?